content/influxdb3/core/visualize-data/tableau.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Use Tableau to visualize data
+list_title: Tableau
+description: >
+  Install and use [Tableau](https://www.tableau.com/) to query and visualize
+  data stored in InfluxDB 3 Core.
+weight: 101
+menu:
+  influxdb3_core:
+    parent: Visualize data
+    name: Tableau
+influxdb3/core/tags: [Flight client, query, flightsql, tableau, sql]
+alt_links:
+  v1: /influxdb/v1/tools/tableau/
+  v2: /influxdb/v2/tools/tableau/
+  cloud: /influxdb/cloud/tools/tableau/
+  cloud-serverless: /influxdb3/cloud-serverless/process-data/visualize/tableau/
+  cloud-dedicated: /influxdb3/cloud-dedicated/process-data/visualize/tableau/
+  clustered: /influxdb3/cloud-clustered/process-data/visualize/tableau/
+metadata: [SQL only]
+draft: true 
```

Comment:
Why are some `draft`?

Reply:
I couldn't get Tableau to connect to InfluxDB. I'm not sure if the issue was on the InfluxDB or Tableau side and didn't have time to troubleshoot. I figured this is something we could come back to in the near future.